### DISCUSSION TITLE
🔒 Hidden Pages

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#769] [New in GitBook]
Take full control over your content with our **Hidden Pages** feature! You can now protect sensitive information by restricting access, ensuring only the right people can view specific pages or entire groups.

🔍 **What can you do with Hidden Pages?**  
- **Restrict access**: Hide critical information behind secure permissions, visible only to authorized users.  
- **Group-level controls**: Manage access at both individual page and group levels for better content security.  

👉 Try it out and see how it works for you! Have any feedback or suggestions? Join the discussion or leave your thoughts—we’re eager to hear from you!

📖 https://docs.gitbook.com/content-editor/editor/content-structure/content-in-a-space#hide-a-page-from-the-table-of-contents

**Comment by denisemcg** [comment]
Hi, am I missing something here? I don't see either of those options you list above. I assume they're only available when the **Hide page** is used alongside other features?
The **Hide page** feature doesn't work as expected nor as described in the user doc.
![image](https://github.com/user-attachments/assets/3a7d3b35-15a5-43fc-a198-3d06bc5fa94e)
I want to hide a page from both the left menu and search results. But it shows up in search results, so it's not really hidden.
![image](https://github.com/user-attachments/assets/580982c0-08d1-4732-bbad-30b260da341d)

**Reply by addisonschultz** [reply]
Hi @denisemcg - you can find these options in “Page options” under visibility - thanks for notifying us, I’ll update the docs!

**Comment by denisemcg** [comment]
Found it! Brilliant, thanks very much.
